impeller/entity/inline_pass_context.cc

```diff
@@ -73,12 +73,7 @@ bool InlinePassContext::EndPass() {
       return false;
     }
   }
-  if (!renderer_.GetContext()
-           ->GetCommandQueue()
-           ->Submit({std::move(command_buffer_)})
-           .ok()) {
-    return false;
```

Comment:
In al these removed calls, there's a bailout before cleanup work is done (e.g. pass_ is held on to).  Should the new EnqueueCommandBuffer() return something other than void (in the fall-through Submit case)?

Reply:
I updated the enqueue to return bool, which can indicate success still for the backends that don't do the batch submit. In this case, its safe to eagerly null out the pass. and the cmd buffer doesn't need to be nulled out since its already nulled from the std::move.